.gitignore

```diff
@@ -380,6 +380,8 @@ tags
 /test/runtime/kbrady/goodAllocSize.good
 /test/runtime/sungeun/chpl-env-gen.h
 /test/runtime/sungeun/chpl-env-gen.test.gen.c
+/test/runtime/panzone/stacktrace.good
+/test/runtime/panzone/fact-stacktrace.good
```

Comment:
I'm pretty sure this .gitignore change should just be reverted. The .good files should be checked in (and you did check them in).